packages/components/src/components/vertical-tab/vertical-tab-content-container.jsx

```diff
@@ -1,16 +1,39 @@
 import classNames from 'classnames';
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
+import { usePrevious } from '../../hooks';
 import Icon from '../icon/icon.jsx';
 
+const SideNotes = ({ side_notes }) => {
+    return (
+        <div className='dc-vertical-tab__content-side-note'>
+            {side_notes?.map((note, i) => (
+                <div className='dc-vertical-tab__content-side-note-wrapper' key={i}>
```

Comment:
(question) Shouldn't we rename the upper div's classname to wrapper? :thinking:

Reply:
should have been dc-vertical-tab__content-side-note-item